src/storer.ts

```diff
@@ -166,12 +132,53 @@ export class Storer {
             }
             this.blobService.createBlockBlobFromText(this.container, blobName, compressedBlobContent, (err) => {
                 if(callback) {
-                    callback(err);
+                    return callback(err);
                 }
+                callback();
             });
-
         });
 
         this.blob = undefined;
     }
+
+    public flushDatabase(callback ? : (err? : Error) => void) : void {
+        const websiteInserts = [];
+        const containsInserts = [];
+        this.websites.forEach(website => {
```

Comment:
Why use ugly `.forEach()` here instead of a `for (let website of this.websites)` loop?